data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-consensual-creation-v1.0.xml

```diff
@@ -0,0 +1,110 @@
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+
+    <changeSet author="R3.Corda" id="ledger-consensual-creation-v1.0">
+
+        <createTable tableName="consensual_transaction">
+            <column name="id" type="VARCHAR(64)">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="privacy_salt" type="VARBINARY(64)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="account_id" type="VARCHAR(64)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created" type="TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="consensual_transaction_status">
+            <column name="transaction_id" type="VARCHAR(64)">
+                <constraints nullable="false" foreignKeyName="fk_transaction_status" references="consensual_transaction(id)"/>
+            </column>
+            <column name="status" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created" type="TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="consensual_transaction_signature">
+            <column name="transaction_id" type="VARCHAR(64)">
+                <constraints nullable="false" foreignKeyName="fk_transaction_signature" references="consensual_transaction(id)"/>
+            </column>
+            <column name="signature_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="signature" type="VARBINARY(64)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created" type="TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addPrimaryKey columnNames="transaction_id, signature_idx"
+                       constraintName="consensual_transaction_signature_pkey"
+                       tableName="consensual_transaction_signature"/>
+
+        <createTable tableName="consensual_transaction_component">
+            <column name="transaction_id" type="VARCHAR(64)">
+                <constraints nullable="false" foreignKeyName="fk_transaction_component" references="consensual_transaction(id)"/>
+            </column>
+            <column name="group_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="leaf_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="data" type="BLOB">
```

Comment:
Can we change any BLOB fields to VARBINARY(1048576)

Reply:
Done! I guess 1 MB will do us for now, but I wonder if we'll run into issues with CPIs in future if they come with a bunch of dependencies bundled together?